app/views/auctions/index.html.erb

```diff
@@ -1,9 +1,9 @@
-<div class="page-header">
+<!-- <div class="page-header">
```

Comment:
Please note that commenting out a section of HTML still runs the ERB within.

Try not to leave commented out code behind. It has a way of growing...